cmd/debug/main.go

```diff
@@ -208,7 +208,7 @@ func generate(c *cli.Context) (err error) {
 				}
 
 				msg++
-				publisher.Publish(&api.Event{
+				publisher.Publish("generator", &api.Event{
```

Comment:
Can you verify this is the correct usage? I didn't run the script to test it.

Reply:
I think that needs to be a topic id - but don't worry about it; we'll need to add authentication etc. Adding "generator" here ensures that ensign builds which is enough for now.